docs/test.ts

```diff
@@ -93,6 +93,18 @@ tracer.init({
   }
 });
 
+tracer.init({
```

Comment:
I don't think this true/false case is strictly neccesary, as we're only testing for types here. The only reason there is a different test for appsec is because it has different possible syntaxes to enable it.

Reply:
Ok, yes. Only one of these should be enough

Reply:
Actually you can directly put it in the first call to `tracer.init()` line 46 ish

Reply:
done